jenkins/Jenkinsfile

```diff
@@ -143,15 +143,15 @@ pipeline {
             description: "A custom image will be build for each of these dataproc versions")
         booleanParam(name: "USE_CUSTOM_DOCKER_IMAGE_VERSIONS", defaultValue: false,
             description: "Check if you want to specify custom image versions below")
-        string(name: "terra_jupyter_base", defaultValue: "0.0.9")
-        string(name: "terra_jupyter_hail", defaultValue: "0.0.9")
-        string(name: "terra_jupyter_r", defaultValue: "0.0.11")
-        string(name: "terra_jupyter_python", defaultValue: "0.0.10")
-        string(name: "terra_jupyter_bioconductor", defaultValue: "0.0.12")
-        string(name: "terra_jupyter_gatk", defaultValue: "0.0.13")
-        string(name: "terra_jupyter_aou", defaultValue: "0.0.1")
-        string(name: "anvil_rstudio_base", defaultValue: "0.0.2")
-        string(name: "anvil_rstudio_bioconductor", defaultValue: "0.0.3")
+        string(name: "terra_jupyter_base", defaultValue: "0.0.13")
+        string(name: "terra_jupyter_hail", defaultValue: "0.0.19")
+        string(name: "terra_jupyter_r", defaultValue: "1.0.5")
+        string(name: "terra_jupyter_python", defaultValue: "0.0.15")
+        string(name: "terra_jupyter_bioconductor", defaultValue: "1.0.5")
+        string(name: "terra_jupyter_gatk", defaultValue: "1.0.5")
+        string(name: "terra_jupyter_aou", defaultValue: "1.0.7")
```

Comment:
I thought we don't ever update the `terra-jupyter-aou` image? is this different than what's in the UI?

Reply:
yea, we're caching both the 0.0.x version and 1.0.x version so that when they're ready to switch, they can do that